### PR TITLE
Add batch version of RowContainer::store API

### PR DESCRIPTION
**Pull request by zhli1142015**

### CODE REVIEW

File: velox/exec/RowContainer.h [hunk 1]
```diff
@@ -302,6 +302,14 @@ class RowContainer {
       char* row,
       int32_t columnIndex);
 
+  /// Stores the first 'size' values from the 'decoded' vector into the
+  /// 'columnIndex' column of 'rows'.
+  void storeVector(
+      const DecodedVector& decoded,
+      const std::vector<char*>& rows,
```
Comment (mbasmanova) [inline velox/exec/RowContainer.h:309]: Can we use folly::Range here and remove 'size' parameter?

Can we rename this method to 'store'?

Comment (zhli1142015) [reply velox/exec/RowContainer.h]: Updated, thanks.

File: velox/exec/RowContainer.cpp [hunk 2]
```diff
@@ -537,6 +537,42 @@ void RowContainer::store(
...
 
+void RowContainer::storeVector(
+    const DecodedVector& decoded,
+    const std::vector<char*>& rows,
+    int32_t size,
+    int32_t column) {
+  VELOX_CHECK_GE(decoded.size(), size);
+  VELOX_CHECK_GE(rows.size(), size);
+  auto numKeys = keyTypes_.size();
+  bool isKey = column < numKeys;
```
[the remark this answers is not in the document]
Comment (zhli1142015) [reply velox/exec/RowContainer.cpp]: Updated, thanks.

File: velox/exec/RowContainer.cpp [hunk 3]
```diff
@@ -537,6 +537,42 @@ void RowContainer::store(
...
+        decoded,
+        size,
+        isKey,
+        rows.data(),
+        offsets_[column]);
+  } else {
+    auto rowColumn = rowColumns_[column];
+    auto offset = rowColumn.offset();
+    auto nullByte = rowColumn.nullByte();
+    auto nullMask = rowColumn.nullMask();
```
[the remark this answers is not in the document]
Comment (zhli1142015) [reply velox/exec/RowContainer.cpp]: Updated, thanks.

File: velox/exec/tests/RowContainerTest.cpp [hunk 4]
```diff
@@ -2063,3 +2063,34 @@ TEST_F(RowContainerTest, hugeIntStoreWithNulls) {
   data->extractColumn(rows.data(), kNumRows, kColumnIndex, extracted);
   assertEqualVectors(source, extracted);
 }
+
+TEST_F(RowContainerTest, storeVector) {
```
[the remark this answers is not in the document]
Comment (zhli1142015) [reply velox/exec/tests/RowContainerTest.cpp]: Updated, thanks.

File: velox/exec/tests/RowContainerTest.cpp [hunk 5]
```diff
@@ -2063,3 +2063,57 @@ TEST_F(RowContainerTest, hugeIntStoreWithNulls) {
   data->extractColumn(rows.data(), kNumRows, kColumnIndex, extracted);
   assertEqualVectors(source, extracted);
 }
+
+TEST_F(RowContainerTest, store) {
+  const uint64_t kNumRows = 1000;
+  auto rowVectorWithNulls = makeRowVector({
+      makeFlatVector<int64_t>(
+          kNumRows, [](auto row) { return row % 5; }, nullEvery(6)),
+      makeFlatVector<StringView>(
```
Comment (mbasmanova) [inline velox/exec/tests/RowContainerTest.cpp:2072]: std::string

**Review by mbasmanova — APPROVED** [review]
@zhli1142015 Looks good. Thanks.

**Comment by zhli1142015** [comment]
@kgpai, could you please help merge this PR?
Thanks.

**Comment by zhli1142015** [comment]
Hello @kgpai  and @bikramSingh91,
Could you please help re-trigger the Meta CI jobs and merge this PR?
Thanks.

**Comment by zhli1142015** [comment]
Thanks @kgpai for helping to trigger the CI , is it ok to merge this PR, or is there any thing from internal CI I need to fix?

**Pull request closed without merging**